IsoCountries.swift

```diff
@@ -15,10 +15,22 @@ struct IsoCountryInfo {
     var calling:String
     var currency:String
     var continent:String
+    var flag: String {
+        return IsoCountries.flag(countryCode: alpha2)
+    }
 }
 
 class IsoCountries {
     
+    class func flag(countryCode: String) -> String {
+        var string = ""
+        var country = countryCode.uppercased()
+        for uS in country.unicodeScalars {
```

Comment:
Can `uS` be renamed to `scalar`. It makes it a bit more clear that is a singular scalar